docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,225 @@
+# Faker connector
+
+The Faker connector generates random data matching a defined structure. It uses
+the [Datafaker](https://www.datafaker.net/) library, which makes the generated
+data more realistic. It can be used to easily populate another data source with
+large and realistic test data, which can help with testing performance of
+applications processing data (like Trino itself), and also application's user
+interface.
+
```

Comment:
After the intro we typically have a Requirements section .. I am not sure if we should have that .. given that there are no requirements ..right?

Reply:
Lets leave it out then.